### PR TITLE
Feature/testing against latest google/cloud releases

### PR DESCRIPTION
**Pull request by TerraSkye**
This pull request allows us to automatically test against the last 2 releases of google/cloud as well against the master branch.

### CODE REVIEW

**Comment by TerraSkye** [comment]
@zoidbergwill could you have a look?

**Comment by zoidbergwill** [comment]
Sorry, I just merged stuff that added conflicts.

**Comment by zoidbergwill** [comment]
Maybe we wanna do 2, 3, and 4, or 1-4, and allow failures of 1? That `cut` idea is hacky but pretty smart. Thanks for the contribution!

**Comment by TerraSkye** [comment]
Yeah i was thinking of that too,
I think its smart to allow failures of 1 and test on 2,3 since thay are the latest 2 releases. But since we have tested for 3 versions. It makes sence to test for 2,3,4

**Comment by TerraSkye** [comment]
I dont know if its possible. But we might need todo a travis retry in before install to detirmene the version?

**Comment by zoidbergwill** [comment]
Yeah, testing on 1-4, and allowing failures for 1 sounds great.

**Comment by zoidbergwill** [comment]
LGTM :+1:

File: .travis.yml [hunk 1]
```diff
@@ -11,23 +11,27 @@ php:
...
 before_install:
   - bash -c 'if [[ "${TRAVIS_PHP_VERSION}" == "hhvm" ]]; then rm phpunit.xml; mv phpunit.hhvm.xml phpunit.xml; fi;'
 
 install:
+  - if [[ -z "$GOOGLE_CLOUD"  ]]; then GOOGLE_CLOUD=$(travis_retry composer show google/cloud -a | grep versions | cut -d "," -f ${RELEASE_TAG} |  sed -e "s/^versions ://" | xargs); fi;
   - travis_retry composer require --no-interaction --no-suggest "google/cloud:${GOOGLE_CLOUD}"
   - travis_retry composer install --no-interaction --no-suggest
 
 script:
+  - export GOOGLE_CLOUD=$GOOGLE_CLOUD
```
Comment (zoidbergwill) [inline .travis.yml]: Is this necessary? I feel like it's a no-op?

**Pull request merged**